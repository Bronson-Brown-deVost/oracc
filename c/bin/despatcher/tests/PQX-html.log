oracc-despatcher: SCRIPT_URL=/hbtin/P382239/html; QUERY_STRING=(null)
404
