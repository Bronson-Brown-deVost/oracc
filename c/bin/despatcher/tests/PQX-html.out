Content-type: text/html; charset=utf-8

Content-type: text/html; charset=utf-8

<?xml version="1.0"?>
<html>
<head>
<link rel="stylesheet" href="/css/wwwhome.css" type="text/css"/>
<title>404 - Oracc Not Found</title>
<base href="http://oracc.museum.upenn.edu/doc/wwwhome/"/></head>
<body class="subpage">

<h1>Oracc URL Not Found.</h1>

<p>You have reached this page because you requested a URL which
Oracc could not understand and which is not a web page on Oracc.</p>

<p>If you were trying to use a special Oracc URL you might find some
help on <a href="http://oracc.museum.upenn.edu/doc/user/projurls/">the
Project URLs page</a>.</p>

<p>Perhaps you mis-typed a project name?  You can see the list of
public projects and access a documentation search
at <a href="http://oracc.museum.upenn.edu">the Oracc home
page</a>.</p>

<!-- Piwik -->
<script type="text/javascript">
var pkBaseURL = (("https:" == document.location.protocol) ? "https://oracc.museum.upenn.edu/piwik/" : "http://oracc.museum.upenn.edu/piwik/");
document.write(unescape("%3Cscript src='" + pkBaseURL + "piwik.js' type='text/javascript'%3E%3C/script%3E"));
</script><script type="text/javascript">
try {
var piwikTracker = Piwik.getTracker(pkBaseURL + "piwik.php", 1);
piwikTracker.trackPageView();
piwikTracker.enableLinkTracking();
} catch( err ) {}
</script><noscript><p><img src="http://oracc.museum.upenn.edu/piwik/piwik.php?idsite=1" style="border:0" alt="" /></p></noscript>
<!-- End Piwik Tag -->
</body></html>
